mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -111,7 +108,6 @@ def test_vhdr_codepage_ansi():
                     line = b'Codepage=ANSI\n'
                 fout.write(line)
     raw = read_raw_brainvision(ansi_vhdr_path, event_id=event_id)
-    data_new, times_new = raw[:]
```

Comment:
I would actually keep it and test something on data_new and times_new as I think this reading part was broken before the test was added